### PR TITLE
AUT-3116/add GitHub workflow for testing

### PR DESCRIPTION
**Pull request by lisocki**
## Description

1. Added Makefile
2. Added GitHub Actions workflow for testing PRs

## Type of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Tests (extending the test suite)
- [ ] Refactor (internal improvement that doesn't change product functionality)
- [ ] Other (if none of the other choices apply)

### CODE REVIEW

**Review by ppodbielski-cloudentity — APPROVED** [review]

**Review by artursmolarek — CHANGES_REQUESTED** [review]
As we discussed, please avoid code duplication and use https://github.com/cloudentity/acp-on-k8s to spin up the cluster.

**Review by mbilski — CHANGES_REQUESTED** [review]
Please use helm testing best practices instead of using the pattern created in acp-on-k8s

File: config/kube-acp-stack.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  ingress:
+    enabled: true
+    hosts:
+      - host: acp.acp-system
+        paths:
+          - path: /
+  features:
+    integration_endpoints: true
+    swagger_ui: true
+  importJob:
```
Comment (jkotiuk) [inline config/kube-acp-stack.yaml:24]: Should we use acp-cd instead legacy import job? @ppodbielski-cloudentity @mbilski

File: config/kube-acp-stack.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+hazelcast:
```
Comment (jkotiuk) [inline config/kube-acp-stack.yaml:1]: we should use redis

**Pull request closed without merging**